### PR TITLE
chore(server,web): bump node version to 20.8

### PR DESCRIPTION
**Pull request by etnoy**
Upgrade node from 18 to 20.8.

Importantly, version 20.8 will be able to run e2e testing without segfaults.

I've tested this manually by uploading images, browsing around etc.

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]
Should we pin it to a hash like before?

**Comment by etnoy** [comment]
> Should we pin it to a hash like before?

I thought about that. Maybe it's enough to stay within the 20.8 tag since it will receive minor updates whenever they come out. Minor versions shouldn't (famous last words) cause any issues, and this is a .0 release which means important fixes might land soon.

File: web/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Our Node base image
-FROM node:18.16.0-alpine3.18@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d as base
+FROM node:20.8-alpine3.18@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d as base
```
Comment (alextran1502) [inline web/Dockerfile:2]: Is the hash correct? It looks similar to the hash of the previous version

Comment (jrasm91) [reply web/Dockerfile:2]: I think this one needs to be removed like the other two

Comment (etnoy) [reply web/Dockerfile]: oops, forgot to remove the hash in the web. fixed now!

**Comment by jrasm91** [comment]
I did some smoke testing by uploading assets, downloading assets, creating albums, shared links, api keys, and everything seems to be working fine.

**Pull request merged**